plugins/amazonq/codetransform/jetbrains-community/src/software/aws/toolkits/jetbrains/services/codemodernizer/ideMaven/MavenRunnerUtils.kt

```diff
@@ -88,7 +88,7 @@ fun runMavenCopyCommands(sourceFolder: File, buildlogBuilder: StringBuilder, log
             telemetryIsCancelled = true
             return MavenCopyCommandsResult.Cancelled
         } else {
-            telemetryErrorMessage = "Maven Copy: bundled Maven failed: exitCode ${copyDependenciesRunnable.isComplete()}"
```

Comment:
Makes sense since we still continue, but is it helpful to have this info emitted to telemetry anyway?

Reply:
I can append it to `reason`. If copy failed but install passed, then `result` will be Succeed, but `reason` will be populated with copy error.